Assets/_BForBoss/_Core/Scripts/EnergySystem/EnergySystemBehaviour.Debug.cs

```diff
@@ -0,0 +1,14 @@
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+namespace BForBoss
+{
+    public partial class EnergySystemBehaviour
+    {
+        public bool UseDebugEnergyCost = false;
+
+        public void SetMaxEnergy()
+        {
+            EnergyData = _energyData.Apply(_energyData.MaxEnergyValue);
```

Comment:
you should write which parameter your setting

Reply:
Updated